apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLogger.cs

```diff
@@ -200,7 +199,7 @@ private Struct CreateJsonPayload<TState>(EventId eventId, TState state, Exceptio
             while (currentLogScope != null)
             {
                 // Determine if the state of the scope are format params
-                if (currentLogScope.State is FormattedLogValues scopeFormatParams)
+                if (currentLogScope.State is IReadOnlyList<KeyValuePair<string, object>> scopeFormatParams)
```

Comment:
This looks like something that could be done in a separate PR - is that the case?

Reply:
The `FormattedLogValues` type has been made `internal` in 3.0. Figured this change could be applied to both the 2.x and 3.0 version.